model.go

```diff
@@ -49,3 +52,11 @@ func (p PullRequestEvent) Issue() Issue {
 func (i Issue) FullName() string {
 	return fmt.Sprintf("%s/%s#%d", i.Repository.Owner, i.Repository.Name, i.Number)
 }
+
+func internalRepositoryRepresentation(githubRepository *github.Repository) Repository {
```

Comment:
Fair point. I think it's fine if I define something like
```go
func HeadRepository(pr *github.PullRequest) Repository {
}
```
will do in the refacto PR